src/dials/command_line/export.py

```diff
@@ -57,26 +57,26 @@
 Examples::
 
   # Export to mtz
-  dials.export models.expt integrated.refl
-  dials.export models.expt integrated.refl mtz.hklout=integrated.mtz
-  dials.export models.expt scaled.refl intensity=scale mtz.hklout=scaled.mtz
+  dials.export integrated.expt integrated.refl
+  dials.export integrated.expt integrated.refl mtz.hklout=integrated.mtz
+  dials.export scaled.expt scaled.refl
+  dials.export scaled.expt scaled.refl intensity=scale mtz.hklout=scaled.mtz
 
   # Export to nexus
-  dials.export models.expt integrated.refl format=nxs
-  dials.export models.expt integrated.refl format=nxs nxs.hklout=integrated.nxs
+  dials.export integrated.expt integrated.refl format=nxs
+  dials.export integrated.expt integrated.refl format=nxs nxs.hklout=integrated.nxs
 
   # Export to mmcif
-  dials.export models.expt integrated.refl format=mmcif
-  dials.export models.expt integrated.refl format=mmcif mmcif.hklout=integrated.mmcif
+  dials.export integrated.expt integrated.refl format=mmcif
+  dials.export integrated.expt integrated.refl format=mmcif mmcif.hklout=integrated.mmcif
 
-  # Export to mosflm
-  dials.export models.expt integrated.refl format=mosflm
+  # Export to mosflm - how is this different to MTZ?
```

Comment:
```suggestion
  # Export to mosflm matrix and instructions files
```
TBH this method is of very little utility though. Perhaps even better to remove it from the examples?